src/types/tokens.rs

```diff
@@ -10,6 +10,38 @@ pub const USDC_BALANCE: FTBalance = FTBalance::with_decimals_and_symbol(6, "USDC
 /// Static instance of [FTBalance] for wNEAR token with correct decimals and symbol.
 pub const W_NEAR_BALANCE: FTBalance = FTBalance::with_decimals_and_symbol(24, "wNEAR");
 
+/// A helper type that represents the fungible token balance with a given precision.
+///
+/// The type is created to simplify the usage of fungible tokens in similar way as the [NearToken] type does.
+///
+/// The symbol is used only for display purposes.
+///
+/// # Examples
+///
+/// ## Defining 2.5 USDT
+/// ```rust
+/// use near_api::USDT_BALANCE;
+///
+/// let usdt_balance = USDT_BALANCE.with_float_str("2.5").unwrap();
+///
+/// assert_eq!(usdt_balance.amount(), 2_500_000);
+/// ```
+///
+/// ## Defining 3 wNEAR using yoctoNEAR
+/// ```rust
+/// use near_api::{W_NEAR_BALANCE, NearToken};
+///
+/// let wnear_balance = W_NEAR_BALANCE.with_amount(3 * 10u128.pow(24));
+///
+/// assert_eq!(wnear_balance.amount(), NearToken::from_near(3).as_yoctonear());
+/// ```
+///
```

Comment:
an api example of comparing two instancs of `FTBalance` with `PartialOrd` can also be added. 
This type may serve as a precise way to compare 2 ft balances itself, that's why a method returning an imprecise float is really not needed.

Non-Derived `PartialOrd` can probably be added; comparing 2 tokens with different symbols doesn't make sense;
comparing 2 tokens with equal symbols but different decimals doesn't make sense either.